docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+
+## Contributing to an Issue
+
+Scan through our existing issues to find one that interests you. You can narrow down the search using labels as filters. See "Label reference" for more information. As a general rule, we assign issues to our internal team only. If you find an issue to work on, you are welcome to open a PR with a fix.
+
+## Contribution workflow
+
+1. [Fork the repository](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo).
+2. Install or update to Node.js, at the version specified in .node-version. For more information consult the [README](../README.md)
+3. Create a working branch and start with your changes!
+
+## Commit Guidelines
+
+When making commits, please adhere to the following guidelines:
+
+- Use the present tense ("Add feature" not "Added feature")
+- Use clear and concise commit messages
+- Reference any relevant issues or pull requests in your commit message using the appropriate keywords (Fixes #issue-number, Resolves #pull-request-number, etc.)
+- Limit the first line to 72 characters or less
+- Make sure each commit represents a logical unit of work
+
+## Pull Requests Guidelines
+
+To ensure a smooth review process and increase the chances of your pull request being merged, please follow these guidelines:
+
+- Ensure that your pull request addresses an existing issue or feature request. If none exists, consider opening one to discuss your proposed changes before submitting the pull request.
+- Provide a clear and descriptive title for your pull request.
+- Include a detailed description of the changes you have made. Explain the purpose and benefits of your changes.
+- Ensure that your code adheres to the project's coding conventions and style guidelines.
+- Include any necessary documentation updates to reflect your changes.
+- Test your changes thoroughly to avoid introducing new bugs.
+- Keep your pull request focused and limited to a single logical change. If you have multiple unrelated changes, consider submitting separate pull requests.
+- Be responsive to any feedback or requests for changes during the review process. Engage in constructive discussions to address any concerns raised by the reviewers.
+- Once your pull request has been approved, it will be merged by the project maintainers.
+
+Thank you for your valuable contributions! 🎉
+
+## Updating documentation
+
+Keeping the documentation up-to-date is essential for maintaining the project's usability and ensuring that users have the necessary information. If you make changes to the codebase, please consider updating the relevant documentation, including the README files.
+
+Make sure the documentation accurately reflects the changes you have made and provides clear instructions or explanations for users. Also, check for any outdated information and remove or update it accordingly.
+
+## Note for Maintainers
```

Comment:
Some of these need to be re-worded. Constructs doesn't make sense in the context of this repo

Reply:
Agreed. Probably needs a bit more time spent on making this specific to our repo.

Reply:
Reworded to remove references to constructs and instead reference packages / tools.